trunk/SUAVE/Methods/Aerodynamics/Common/Higher_Fidelity/Drag/untrimmed.py

```diff
@@ -1,12 +1,14 @@
-## @ingroup Methods-Aerodynamics-SU2_Euler
+## @ingroup Methods-Aerodynamics-Higher_Fidelity-Drag
```

Comment:
I did but in this one for example it looks like it's just a matter of when items are added.